docs/build-with-powerloom/snapshotter-node/lite-node-v2/build-sh-advanced.md

```diff
@@ -1,5 +1,6 @@
 ---
 sidebar_position: 3
+title: "CLI: Advanced Usage and Recipes"
```

Comment:
This could be a bit misleading with the imminent release of the unified CLI for snapshotters.